pyhive/tests/test_hive.py

```diff
@@ -72,6 +72,21 @@ def test_complex(self, cursor):
             '0.1',
         ]])
 
+    @with_cursor
+    def test_async(self, cursor):
+        cursor.execute('SELECT * FROM one_row', async=True)
+        while cursor.poll() != ttypes.TOperationState.FINISHED_STATE:
+            cursor.fetch_logs()
+        
+        self.assertEqual(len(cursor.fetchall()), 1)
+    
+    @with_cursor
+    def test_cancel(self, cursor):
+        cursor.execute('SELECT * FROM one_row', async=True)
+        cursor.cancel()
```

Comment:
`select reflect("java.lang.Thread", "sleep", 10 * 1000) from one_row` ought to do it.
Or if that doesn't work, `select count(*) from one_row` will be slower because it triggers a MR job`